react-rust-postgres/backend/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM jdrouet/rust-nightly:buster-slim AS base
```

Comment:
Is it possible to use an official image instead of your own? Is there any specific reason to use a custom nightly image?

Reply:
we need the nightly version and this image is the only one doing multiarch (do we want people to be able to run it on their raspberry pi? I think...).

Reply:
Sure! Thanks for the explanation